.travis.yml

```diff
@@ -1,35 +1,39 @@
 language: python
-
 python:
-  - "3.6"
-  - "3.7"
-
+- '3.6'
+- '3.7'
 dist: xenial
 sudo: false
-
+addons:
+  ssh_known_hosts: shrimp.octet.services
 services:
 - docker
-
 before_install:
-  - git clone https://github.com/DEX-Company/barge.git
-  - cd barge
-  - git checkout tags/dex-2019-02-01
-  - bash -x start_ocean.sh --no-brizo --no-pleuston --local-spree-node 2>&1 > start_ocean.log &
-  - cd ..
-  - sleep 15
-
+- git clone https://github.com/DEX-Company/barge.git
+- cd barge
+- git checkout tags/dex-2019-02-01
+- bash -x start_ocean.sh --no-brizo --no-pleuston --local-spree-node 2>&1 > start_ocean.log
```

Comment:
I'm not clear on how we can view the start_ocean.log on travis, e.g.
https://travis-ci.com/DEX-Company/ocean-py
is there a way we can view that log file?

Reply:
I'm not sure either, I'll have a look at travis to see if we can get the log file after running the job